### PR TITLE
Autogenerate Version=<git revision>-<build time>

### PR DESCRIPTION
**Pull request by gshilin-sdb**
For example: 
```
 $ ./ocm-support version
a39f326-11/16/22@12:35:02
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,11 +3,19 @@ OUTPUT_DIR :=_output
 # Constants
 GOPATH := $(shell go env GOPATH)
 
+# Version
+revision:=$(shell git rev-parse --short HEAD)
+build_time:=$(shell date +%D@%T)
+version:=$(revision)-$(build_time)
```
Comment (cristianoveiga) [inline Makefile:9]: I like the idea of having the version set by the git revision and build time, but I don't want to lose the semantic version info (0.1.0). In my opinion, that's very user friendly and can allows for quick inspection/verification.

I would suggest having it as something like: 

version:=$(semantic-version)-$(revision)-$(build_time)

What do you think?

Comment (gshilin-sdb) [reply Makefile:9]: added

File: Makefile [hunk 2]
```diff
@@ -3,18 +3,27 @@ OUTPUT_DIR :=_output
...
+sem_version:=$(SEMVERSION)
```
Comment (cristianoveiga) [inline Makefile:10]: where is the SEMVERSION defined?

File: cmd/ocm-support/version/cmd.go [hunk 3]
```diff
@@ -18,5 +18,5 @@ var Cmd = &cobra.Command{
 }
 
 func run(cmd *cobra.Command, argv []string) {
-	_, _ = fmt.Fprintf(os.Stdout, "%s\n", info.Version)
+	_, _ = fmt.Fprintf(os.Stdout, "%s\n%s\n", info.Version, info.VersionStamp)
```
Comment (cristianoveiga) [inline cmd/ocm-support/version/cmd.go:21]: Nitpick:

```
0.1.0
de45724-11/16/22@10:35:46
```
```
0.1.0 (build de45724-11/16/22@10:35:46)
```

**Comment by cristianoveiga** [comment]
lgtm

**Pull request merged**